Task_9/Main.java

```diff
@@ -0,0 +1,48 @@
+package Task_9;
+import java.io.*;
+
+/**
+ * Class has a method "main", which is starting point in the program and method "writeToFile",
+ * which write information into file.
+ * @author Bogatko Irina
+ * @version 1.0
+ * @since 18.10.2016
+ */
+public class Main {
+
+    /**
+     * Calls methods, which create html-table and add in this table information about name, type,
+     * date of creation and size of directories or files.
+     * @param args command-line params
+     */
+    public static void main (String[] args) {
+        File file = new File(System.getProperty("user.dir"));
+        HtmlBuilder html = new HtmlBuilder();
+        File[] fileList = file.listFiles();
+
+        for (File fileWithTable : fileList) {
+            GetItem inform = new GetItem(fileWithTable);

```

Comment:
ok, here you called object inform - we are on the same way that it's actually information about something. So let rename the GetItem as it was suggested